torchmd/parameters.py

```diff
@@ -159,8 +160,8 @@ def build_parameters(self, ff, mol, terms):
     # def make_charges(self, ff, atomtypes):
     #     return torch.tensor([ff.get_charge(at) for at in atomtypes])
 
-    def make_masses(self, ff, atomtypes):
-        masses = torch.tensor([ff.get_mass(at) for at in atomtypes])
+    def make_masses(self, elements):
+        masses = torch.tensor([periodictable[el].mass for el in elements])
```

Comment:
I believe if get_mass returns a valid mass it should take precedence over periodictable.

Reply:
Not sure about it, probably @stefdoerr knows better.

Reply:
Yes, better to take first the FF mass if it exists. Not certain but it could be that we are affected by hydrogen mass repartitioning otherwise.